Hunting Queries/LAQueryLogs/QueryForSecrets.yaml

```diff
@@ -0,0 +1,24 @@
+id: 2bf19f27-0466-4c16-a821-ce84e524476d
+name: Queries looking for secrets
+description: |
+  'This hunting query looks for queries that appear to be looking for secrets or passwords in tables.'
+requiredDataConnectors:
+  - connectorId: CustomConnector
+    dataTypes:
+      - LAQueryLogs
+tactics:
+  - Collection
+relevantTechniques:
+  - T1530
+  - T1213
+query: |
+
+  let keywords = dynamic(["password", "pwd", "creds", "credentials", "secret"]);
```

Comment:
the "password" keyword match hits on too many records as it catches OperationName items such as - "Change user password"  We should validate we are not capturing this type of thing.